backend/src/main/java/edu/netcracker/backend/model/state/trip/Archived.java

```diff
@@ -0,0 +1,26 @@
+package edu.netcracker.backend.model.state.trip;
+
+import edu.netcracker.backend.model.Trip;
+import edu.netcracker.backend.model.User;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.List;
+
+@Component
+public class Archived extends TripState {
+
+    private final static int value = 5;
```

Comment:
It's difficult to understand what this variable does. I suggest changing name. In other cases below all the same

Reply:
Renamed to databaseValue.